.github/workflows/publish.yml

```diff
@@ -11,15 +11,6 @@ on:
         type: string
         required: false
         default: 8
-    secrets:
```

Comment:
Just need to remove these secrets from README 😉

Reply:
Ah yes. The documentation... 😉